src/components/video.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import YouTube from 'react-youtube';
+import * as Icon from 'react-feather';
+import Card from './card';
+
+// type VideoProps = {
+//   title: string,
+//   url: string,
+//   id: string,
+// };
+
+const sampleData = {
+  title: 'Rick Astley - Never Gonna Give You Up (Video)',
+  author: 'Official Rick Astley',
+  yid: 'dQw4w9WgXcQ',
+  spotify: 'https://open.spotify.com/track/4uLU6hMCjMI75M1A2tKUQC',
+  likes: 27,
+  genres: ['soul', 'pop'],
+  queue: [
+    'zReP_EYZGEw',
+    'dQw4w9WgXcQ',
+    'PJWemSzExXs',
+    'k4V3Mo61fJM',
+    'e-ORhEE9VVg',
+    '2hlT8CqZ2pA',
+    'U0V1y2p1sgs',
+  ],
+};
+
+export default function Video(props) {
+  let preQueue;
+  if (props.queue) {
+    preQueue = props.queue;
+  } else {
+    preQueue = sampleData.queue;
+  }
+  let [queue, changeQueue] = useState([
+    props.id || sampleData.yid,
+    ...preQueue,
+  ]);
+
+  useEffect(() => {
+    if (document !== undefined) {
+      document.body.scrollTop = 0; // For Safari
+      document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
+    }
+  }, []);
+
+  return (
+    <div className='video'>
+      <div className='video-container'>
+        <YouTube
+          videoId={queue[0]}
+          onEnd={() => {
+            if (queue.length > 0) {
+              changeQueue(queue.slice(1));
+            }
+          }}
+          opts={{
+            height: '550',
+            width: '900',
+            playerVars: {
+              autoplay: 1,
+            },
+          }}
+        />
+
+        <div className='desc'>
+          <div className='text'>
+            <h1 className='title'>{sampleData.title}</h1>
+            <h2>{sampleData.author}</h2>
+          </div>
+
+          <div className='widgets'>
+            <div className='icons'>
+              <a href={'https://www.youtube.com/watch?v=' + sampleData.yid}>
+                <Icon.Youtube></Icon.Youtube>
+              </a>
+              <a href={sampleData.spotify}>
```

Comment:
This is such a sweet addition! Will make sure the API returns this if it does have the data.
Can you add the `target="_blank" rel="noopener noreferrer"` attributes so the link opens in a new tab rather than taking the user away from our site?

Reply:
Thanks for pointing that out, fixed it in the new commit